src/observability/observabilityTypes.ts

```diff
@@ -23,6 +23,7 @@ export type TransactionObservabilityManager = {
   /**
    * Ends the transaction
    * @param uniqueTransactionKey - used for identifying specific ongoing transaction. Must be reasonably unique to reduce possibility of collisions
+   * @param wasSuccessful - indicates if the transaction was successful or not
```

Comment:
should we default to true?

Reply:
I thought about the default, the advantage of the optional value is that it lets different implementations define what means "default value" instead of forcing them to have true.

But I am not really sure if it has more const (or more important ones) than pros, so please feel free to disagree and I will change it, no strong opinion from my side

Reply:
fair point, it should be handled on an implementation side